### DISCUSSION TITLE
Issues viewing capsule in every client EXCEPT https://portal.mozz.us/ web proxy

### DISCUSSION THREAD

**Discussion by AlexBIrvine** [mbrubeck/agate#166] [Q&A]
I setup my Agate server following the instructions on https://landchad.net/gemini.  This is running on a VPS from vultr, debian based server.  

I'm able to view my .gmi file and certificate on this web proxy:  gemini://6pakal.com/ (https://portal.mozz.us/gemini/6pakal.com/)
However, when attempting to view the same site on either Lagrange or Amfora, it says it cannot connect due to network connection error (vague error).  

I ran the server stress test script found here ( https://github.com/michael-lazar/gemini-diagnostics ) and all tests failed.  
The DNS is resolving to the proper IPv6 address.  

I'm not sure what else to troubleshoot from here, or what more information would be useful.  I'm pretty new to server side computing.  
Any help is appreciated. Below is the configuration file I used.  
[config.txt](https://github.com/mbrubeck/agate/files/8403503/config.txt)

**Comment by Johann150** [comment]
Since your server only has an IPv6 address record in DNS, the problem might come from your local network not having IPv6 enabled/only having IPv4 connectivity.

I tested from a network with only IPv4 connectivity and could not connect to your server at all. I tested the same from a network with IPv6 connectivity and it seemed to work.
